### DISCUSSION TITLE
Genmon Power Disconnect

### DISCUSSION THREAD

**Discussion by spatel7786** [jgyates/genmon#1048] [Questions] [answered]
Hello @jgyates @skipfire ,

I have my Pi, Hat & splitter all setup and working flawlessly along with keeping the MobilLink intact and functioning. I also have my email notification setup and receiving them. I would like to place the Pi/Hat/splitter in a weatherproof enclosure to keep it protected from the elements. Is there a proper procedure to shutdown the device and keep from losing all of the settings that I have saved? Or can I simply just disconnect the barrel battery connector that powers the HAT and disconnect from the controller and remove everything to install in a enclosure? I saw the "shutdown" command in the advance settings screen on the web interface of genmon so it has me worried about pulling the power cord.

**Comment by skipfire** [comment, marked as answer]
Ideally you would use the shutdown command, but it is doubtful that just pulling the plug will hurt anything.  If you have Log2Ram installed (which the preloaded units have for a couple months if you got a preloaded unit) pulling the plug will lose data since the last write (default is I believe hourly but not necessarily on the hour).  Is there any reason you want to move the splitter to the enclosure instead of leaving it inside the generator?  Moving the splitter will require a notably larger box and routing the MobileLink cable.

**Reply by jgyates** [reply]
hi @spatel7786 
The shutdown command on the advanced settings page will shut down the operating system (with the console shutdown command). But I also agree with @skipfire that if you have log2ram and genmon is not recording events like a power outage, then just pulling power is relatively harmless.

**Reply by spatel7786** [reply]
@skipfire @jgyates  - I do not have the preloaded unit. I am using my own Pi 3b+ and I purchased the standard HAT and splitter from pintsize.me. Currently all 3 items are sitting inside the battery compartment of the generator by the louvers just hanging. I want to place everything inside of a plastic weatherproof enclosure in the exact same location above the battery as debris and dirt tends to accumulate around the battery area. Trying to keep it all nice and clean lol. Everything will still remain in the generator. Since I do not have the preloaded unit, am I safe to unplug everything and not worry about a "proper shutdown" method?

**Reply by jgyates** [reply]
To be 100 percent safe, click the shutdown button and wait 30 seconds to remove power. Shutting down this way helps protect the file system from errors. If you have not installed log2ram installed the shutdown button is a better way to go than just pulling power.

**Reply by jgyates** [reply]
FYI, the shutdown button just calls this command on the console:

      sudo shutdown -h now

The -h is poweroff and the time to shutdown is now.

**Reply by spatel7786** [reply]
@jgyates @skipfire Thank you both for all of the information on this! I will use the shutdown command only. Awesome software and hardware guys!

**Reply by AlexOu812** [reply]
hi @spatel7786,  
I have LTE Mobile Link with my generator and just completed setting up my Genmon device.   I have the hat from pintsize.me along with the 8 conductor molex harness.  Like you, I want to keep Mobile Link connected.  Even though my 1 year free subscrition has expired, it looks like data is still being provided to Generac or the company that installed my unit.  Because of that, I would still like to keep the cell phone connected to the control boad while using Genmon.  I see you are using the splitter board but think I might be able to modify the harness if in fact you are using LTE Mobile Link and not WiFi Mobile Link.

Thanks,
Alex

**Reply by skipfire** [reply]
@AlexOu812 the LTE I have seen uses RS232 and that would conflict with GenMon. If it uses RS485 then my splitter or a hand built one could work.

**Reply by spatel7786** [reply]
> hi @spatel7786, I have LTE Mobile Link with my generator and just completed setting up my Genmon device. I have the hat from pintsize.me along with the 8 conductor molex harness. Like you, I want to keep Mobile Link connected. Even though my 1 year free subscrition has expired, it looks like data is still being provided to Generac or the company that installed my unit. Because of that, I would still like to keep the cell phone connected to the control boad while using Genmon. I see you are using the splitter board but think I might be able to modify the harness if in fact you are using LTE Mobile Link and not WiFi Mobile Link.
> 
> Thanks, Alex

I’m using the WiFi mobile link and not LTE.
